# ReceiverTime block
# ROS message header
std_msgs/Header header

# SBF block header including time header
BlockHeader  block_header

int8  utc_year
int8  utc_month
int8  utc_day
int8  utc_hour
int8  utc_min
int8  utc_second
int8  delta_ls
uint8 sync_level